src/util/Errors.js

```diff
@@ -63,6 +63,13 @@ function WebauthnAbortError() {
 }
 WebauthnAbortError.prototype = new Error();
 
+function FlowError(message, flowSetting) {
```

Comment:
`FlowError` is very generalized.. doesn't say that this is only being used in a custom widget not hosted on okta..